---
published: false
---
## A New Post

Enter text in [Markdown](http://daringfireball.net/projects/markdown/). Use the toolbar above, or click the **?** button for formatting help.

## InterPro and Intrinsically Disordered Proteins
### Structure is more conserved than sequence
As Linus Pauling, a double Nobel laureate ([Peace 1962](https://www.nobelprize.org/prizes/peace/1962/pauling/lecture/) and [Chemistry 1954](https://www.nobelprize.org/prizes/chemistry/1954/pauling/lecture/)), stated, "If we want to understand cells, we must know their structure in terms of molecules" [[1](https://www.youtube.com/watch?v=7tevU4Cu4XE)]. With this purpose in mind, to grasp the physiology of living organisms, we study proteins at the molecular level. To illuminate our path, we rely on the potent light shed by the evolutionary process, as nothing in biology makes sense except in the light of evolution [[2](https://doi.org/10.2307/4444260)].
Throughout the billions of years in the evolutionary history of life on Earth, protein structures have been better preserved than their amino acid sequences [[3](https://europepmc.org/article/MED/10195279), [4](https://europepmc.org/article/MED/3709526)]. Therefore, the identification of distant evolutionary relationships between proteins is considerably more effective when comparing structures rather than when comparing their sequences [5, 6, 7]. In fact, due to the recent surge in high-quality structural predictions, the identification of new protein domains and the precise determination of their boundaries have been accelerated through the millions of structural models generated by artificial intelligence techniques [8, 9, 10, 11].

### Challenges in analysing disordered proteins and tools for assessment in InterPro
There are instances when protein structures, which are ultimately responsible for the protein functionality, present challenges in their analysis, prediction, or experimental characterisation. A significant proportion of proteins or protein regions exist in a disordered state when considered in isolation, which accounts for approximately one-third of the human proteome [12]. This disordered state can appear as either a lack of a stable structure throughout the entire sequence or as specific regions within the protein, known as intrinsically disordered proteins (IDPs) or intrinsically disordered regions (IDRs), respectively [13]. In these proteins, comparing the structural form of their monomeric state with other proteins does not enable us to identify structural similarities that would allow us to infer homology.
Using InterPro, we can easily determine whether our protein of interest is an IDP or rich in IDRs [14]. Recently, the DisProt database, a major manually curated repository of Intrinsically Disordered Proteins [15], has been integrated into InterPro. Additionally, in InterPro, we can map the output of MobiDB-lite (a consensus disorder prediction tool) [16] and the predicted local distance difference test (pLDDT) confidence scores from AlphaFold [17], these scores provide information about structural stability at the residue level, with a pLDDT value below 70 indicating potential lack of stable structure in isolation (cartoons coloured in yellow and red in **Figures 1 and 3**).

An example of such proteins, rich in IDRs, can be found in the Yap/Taz/FAM181/PERCC1 family in humans (**Figure 1**). Several members of this family have been experimentally described as regulators of transcriptional activity by interacting with TEAD (TEA/ATTS domain) transcription factors, which serve as the main nuclear effectors of the Hippo pathway, crucial in regulating organ size and maintaining tissue homeostasis in animals. The structural similarity among Yap/Taz/FAM181/PERCC1 family members becomes evident only when we examine these structures while they interact with the proteins they specifically bind to, the TEAD transcription factors. However, their recognition as homologues was only achievable through the identification of conserved regions in their amino acid sequences, using classical computational tools for identifying statistically significant sequence similarity based on the comparison of profiles against sequences ([HMMER](http://hmmer.org/)) and profiles against profiles ([HHpred](https://toolkit.tuebingen.mpg.de/tools/hhpred)) (**Figure 2**) [[18](https://europepmc.org/article/MED/36699391)].

![idr_luis_f1.png]({{site.baseurl}}/assets/media/images/posts/idr_luis_f1.png)

**Figure 1. AlphaFold predicted structure and protein sequence viewer of the AlphaFold pLDDT score and MobiDB-lite prediction for PERCC1 and FAM181B.**
The figure displays the AlphaFold models (cartoons coloured by pLDDT confidence score) and the MobiDB-lite predictions in the protein sequence viewer as displayed on the InterPro website for two human paralogues (PERCC1 and FAM181B proteins) rich in IDRs.


![idr_luis_f2.png]({{site.baseurl}}/assets/media/images/posts/idr_luis_f2.png)

**Figure 2. Representative multiple sequence alignment of two consecutive TEAD interaction motifs (‘interfaces 2 and 3’) in Yap/Taz/FAM181/PERCC1 family.** Protein subfamilies are indicated by coloured bars at the left of the alignment: PERCC1, FAM181B and YAP/TAZ are indicated in red, yellow and purple, respectively. Limits of protein sequences included in the alignment are indicated by flanking residue positions. Numbers inside green boxes represent excised unconserved sequence. The alignment was generated with T-Coffee and presented with the program Belvu using a colouring scheme indicating the average BLOSUM62 scores (which are correlated with amino acid conservation) of each alignment column: red (>3), violet (between 3 and 0.8) and light yellow (between 0.8 and 0.2). Sequences are named according to their UniProt identification.

Another example is the family of histone chaperones SPT2, which is widely distributed among eukaryotes. Their conserved regions correspond to randomly distributed α-helices  (Figure 3), and their homology is impossible to deduce through the comparison of their structures (see video at: [https://youtu.be/WqaDEgl4VeY](https://youtu.be/WqaDEgl4VeY)). In this case, only sequence conservation analysis enables us to establish orthology relationships and define this family [[19](https://doi.org/10.1101/2023.02.16.528451)].


![idr_luis_f3.png]({{site.baseurl}}/assets/media/images/posts/idr_luis_f3.png)

**Figure 3. AlphaFold predicted structure and protein sequence viewer of the AlphaFold pLDDT score and MobiDB-lite prediction for humans and mouse SPT2.**
The figure displays the AlphaFold models (cartoons coloured by pLDDT confidence score) and the annotations found in InterPro for two SPT2 orthologues (human and mouse SPT2 proteins) rich in IDRs.

### To bring order to disorder... protein sequence conservation
Despite having a wealth of high-quality protein structural data, both experimental (Protein Data Bank) and computationally predicted, such as those produced by AlphaFold and Meta AI [[17](https://europepmc.org/article/MED/34265844), [20](https://europepmc.org/article/MED/36420884), [21](https://europepmc.org/article/MED/36927031)], in the case of IDPs and IDRs, we won't be able to establish homology relationships through the comparison of their known or predicted monomeric structures. 
To shed light on this "Dark Proteome", the identification of protein sequence conservation will continue to be essential for understanding, classifying, and deciphering IDRs roles. The databases of sequence conservation-based protein alignments, such as Pfam, Smart, or CDD, integrated into InterPro, play a pivotal role and will continue to be highly valuable tools in the task of predicting protein functions for proteins rich in IDRs.
InterPro provides a comprehensive view, facilitating the identification of protein regions where disorder and evolutionary conservation overlap - areas that, despite their predicted unstable structure, likely hold significant functional relevance [[22](https://doi.org/10.1038/s41580-023-00673-0)].








