---
published: true
layout: post
category: interpro
image_category: website
author: Typhaine Paysan-Lafosse
---
This release includes new features and improvements to the InterPro website. The list of changes is detailed below. If you have any feedback or suggestions, please [contact us](https://www.ebi.ac.uk/support/interpro).

## Content
In this release, we have updated Pfam to version 36.0 and updated InterPro entries accordingly. For more information about the changes in Pfam 36.0, you can have a look at [this blog post](https://xfam.wordpress.com/2023/09/18/pfam-36-0-release/).

We also have integrated 1,127 new signatures from the Pfam (1,048), CDD (40), CATH-Gene3D (25) and PANTHER (9) databases.

## Protein sequence viewer updates
### Single domain viewer
A feature found in the Pfam website that was missing in InterPro was to be able to visualise the different domains found in a protein in one line (Figure 1A). We have implemented this feature in the protein sequence viewer on the protein page, as shown in Figure 1B. Please note that this representation is generated automatically using the type of the member databases models, which might differ from the InterPro entries types.

![interpro_96_pfam_vs_interpro_single_dom.png]({{site.baseurl}}/assets/media/images/posts/interpro_96_pfam_vs_interpro_single_dom.png)
Figure 1. Pfam (A) versus InterPro (B) representative domains view for [Q7X923](https://www.ebi.ac.uk/interpro/protein/UniProt/Q7X923/).

### Tidying up Other features
In the _Other features_ section of the protein sequence viewer, the tracks are now sorted by source database, so all the matches from the same resource are grouped together as shown in Figure 2. Additionally, all the annotations have the source database as part of the label, making it clear where the annotations are coming from.

![interpro_96_other_features_95_vs_96.png]({{site.baseurl}}/assets/media/images/posts/interpro_96_other_features_95_vs_96.png)
Figure 2. Other features section in the protein sequence viewer in InterPro 95.0 (A) versus InterPro 96.0 (B) for [Q7X923](https://www.ebi.ac.uk/interpro/protein/UniProt/Q7X923/).

## New features
### Addition of AntiFam pages
In InterPro release 88.0 (March 2022), we announced the addition of AntiFam as part of the _Other features_ annotations displayed in the protein sequence viewer. To increase the visibility of AntiFam annotations, we have decided to add AntiFam in the _Browse By Member DB_ (Figure 3A) and create signature pages for each entry (Figure 3B). Additionally, we have moved the AntiFam annotations to the _Unintegrated_ category in the protein sequence viewer (Figure 3C,D). Furthermore, the AntiFam matches are accessible from the _Entries_ tab in the protein page.

![interpro_96_antifam_88_vs_96.png]({{site.baseurl}}/assets/media/images/posts/interpro_96_antifam_88_vs_96.png)
Figure 3. Browse functionality including AntiFam (A). AntiFam signature page (B) AntiFam matches for [A0A011PAP6](https://www.ebi.ac.uk/interpro/protein/UniProt/A0A011PAP6/) between InterPro 88.0 (B) and InterPro 96.0 (C).

### Display of structure chains without proteins
PDB structures chains are often associated with UniProt proteins, but not always. On structure pages, we were previously only displaying chains that were associated with proteins. We now display all the chains, whether they are associated with a protein or not.

Additionally, the _Entries_ and _Proteins_ tab in the Structure page and the _Structure_ tab in other pages (e.g. InterPro entry page, Protein page) now display the structure sequence, and a match track for each chain for each structure match, as shown in Figure 4.

![interpro_96_proteins_in_struct.png]({{site.baseurl}}/assets/media/images/posts/interpro_96_proteins_in_struct.png)
Figure 4. List of InterPro entries and location of the chains matches for the PDB structure [4nxu](https://www.ebi.ac.uk/interpro/structure/PDB/4nxu/entry/InterPro/#table).

## Migration of the InterPro codebase to TypeScript and Visual framework 2.0
As mentioned in a previous blog post, we have started the process to move the InterPro website code base to use TypeScript instead of JavaScript+FlowJS, and we are also migrating the codebase from using the EBI visual framework 1.4 (with foundation) to using Visual framework 2.0 (Vanilla CSS). 
In this release, we have migrated the following pages and components:
- The Protein page Overview
- The InterPro entry page Overview
- The Signature page Overview
- The Structure page (Overview, Entries, Proteins)
- The Taxonomy, Set and Proteome Overview
